apiserver/facades/client/spaces/spaces_test.go

```diff
@@ -900,45 +907,25 @@ func (s *LegacySuite) TestShowSpaceError(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, "getting environ: retrieving model config: boom")
 }
 
-func (s *LegacySuite) TestCreateSpacesModelConfigError(c *gc.C) {
-	defer s.setupMocks(c).Finish()
-	s.makeAPI(c)
-
-	apiservertesting.SharedStub.SetErrors(
-		errors.New("boom"), // Backing.ModelConfig()
-	)
-
-	args := params.CreateSpacesParams{}
-	_, err := s.facade.CreateSpaces(stdcontext.Background(), args)
-	c.Assert(err, gc.ErrorMatches, "getting environ: retrieving model config: boom")
-}
-
 func (s *LegacySuite) TestCreateSpacesProviderOpenError(c *gc.C) {
```

Comment:
This test is different now. I think we should update the name of the test and write a small comment above to explain what it is that we are trying to prove here with the test. Tests are about maintaining functional contracts. What is the contract we want to see maintained?

Reply:
I changed the test name slightly and added a comment, but IMO this test is more or less the same